src/main/java/tr/ogu/edu/school/schoolapp/controller/ActTicketController.java

```diff
@@ -0,0 +1,29 @@
+package tr.ogu.edu.school.schoolapp.controller;
+
+import java.io.FileNotFoundException;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import tr.ogu.edu.school.schoolapp.model.ActTicket;
+import tr.ogu.edu.school.schoolapp.service.ActTicketService;
+
+@RestController
+@RequestMapping("/act-tickets")
+public class ActTicketController {
+    private final ActTicketService actTicketService;
+
+    // Constructor
+    public ActTicketController(ActTicketService actTicketService) {
+        this.actTicketService = actTicketService;
+    }
```

Comment:
Constructor'ı yazmak yerine clean-code için lombok kullanıyoruz ve sınıfın üzerine örneğin @RequiredArgsConstructor ekliyoruz.